app/src/main/java/ru/vssemikoz/newsfeed/utils/ActivityUtils.java

```diff
@@ -0,0 +1,20 @@
+package ru.vssemikoz.newsfeed.utils;
+
+import androidx.annotation.NonNull;
+import androidx.fragment.app.Fragment;
+import androidx.fragment.app.FragmentManager;
+import androidx.fragment.app.FragmentTransaction;
+
+import static androidx.core.util.Preconditions.checkNotNull;
+
+public class ActivityUtils {
+
+    public static void addFragmentToActivity(@NonNull FragmentManager fragmentManager,
+                                             @NonNull Fragment fragment, int frameId) {
+        checkNotNull(fragmentManager);
+        checkNotNull(fragment);
+        FragmentTransaction transaction = fragmentManager.beginTransaction();
+        transaction.add(frameId, fragment);
```

Comment:
You're adding your fragment using `frameId` variable which is not the best name as we need a container. A `containerId` looks like a better name for this variable. What do you think?